pkg/internal/export/json/json.go

```diff
@@ -0,0 +1,239 @@
+// Package json provides an export node to print span contents as JSON objects
+package json
```

Comment:
Please add tests for each function of this file.

Reply:
Done